docs/Quick-Start-Guide/Architecture.md

```diff
@@ -17,10 +17,10 @@ Gluster file system supports different
 types of volumes based on the requirements. Some volumes are good for
 scaling storage size, some for improving performance and some for both.
 
-1.__ **Distributed Glusterfs Volume** __- This is the type of volume which is created by default if no volume type is specified.
+1.\*\* **Distributed Glusterfs Volume** **- This is the type of volume which is created by default if no volume type is specified.
```

Comment:
This would appear as ** Distributed Dispersed Glusterfs Volume**
Not sure if ** are required besides

Reply:
For some reason, prettier thought it would be cool to add them :P

Fixed it everywhere.

Regards